modules/casper.js

```diff
@@ -1498,8 +1498,9 @@ Casper.prototype.resourceExists = function resourceExists(test) {
             break;
         case "function":
             testFn = test;
-            if (phantom.casperEngine !== "slimerjs")
-                testFn.name = "_testResourceExists_Function";
+            // if (phantom.casperEngine !== "slimerjs") {
```

Comment:
In this point was problem with bug in Phantom, not Slimer. The code is not removed, because I need your opinion.

After this edit, test passed in both engines.